boards/arm/96b_argonkey/96b_argonkey_defconfig

```diff
@@ -23,6 +23,21 @@ CONFIG_I2C=y
 # enable SPI
 CONFIG_SPI=y
 
+#enable sensors
+CONFIG_SENSOR=y
+CONFIG_HTS221=y
+CONFIG_LPS22HB=y
+CONFIG_LSM6DSL=y
+CONFIG_LSM6DSL_SPI=y
+CONFIG_LSM6DSL_SPI_MASTER_DEV_NAME="SPI_2"
+CONFIG_LSM6DSL_SPI_SELECT_SLAVE=1
+CONFIG_LSM6DSL_TRIGGER_GLOBAL_THREAD=y
+CONFIG_LSM6DSL_TRIGGER=y
+CONFIG_LSM6DSL_GPIO_DEV_NAME="GPIOB"
+CONFIG_LSM6DSL_GPIO_PIN_NUM=1
+CONFIG_LSM6DSL_SENSORHUB=y
+CONFIG_LSM6DSL_EXT0_LIS2MDL=y
```

Comment:
I am not familiar with 96b_argonkey but shouldn't all the following being moved in the 96b_argonkey's test sample? 
```
CONFIG_SENSOR=y
CONFIG_HTS221=y
CONFIG_LPS22HB=y
CONFIG_LSM6DSL=y
CONFIG_VL53L0X=y
```
And the following in the board's Kconfig.defconfig?
```
CONFIG_LSM6DSL=y
CONFIG_LSM6DSL_SPI=y
CONFIG_LSM6DSL_SPI_MASTER_DEV_NAME="SPI_2"
CONFIG_LSM6DSL_SPI_SELECT_SLAVE=1
CONFIG_LSM6DSL_TRIGGER_GLOBAL_THREAD=y
CONFIG_LSM6DSL_TRIGGER=y
CONFIG_LSM6DSL_GPIO_DEV_NAME="GPIOB"
CONFIG_LSM6DSL_GPIO_PIN_NUM=1
CONFIG_LSM6DSL_SENSORHUB=y
CONFIG_LSM6DSL_EXT0_LIS2MDL=y
```

Reply:
@ydamigos  Exact. Configuration should be done at board level (use `Kconfig.defconfig` as much as possible).
Then, enabling the sensors (SENSOR, LSM6DSL, ...) should be done at application level (`sample/.../.../prj.conf` in this case)

Reply:
Thx guys!
I will follow your indications.